### PR TITLE
Week3 Lesson1.3 Submission

### PR DESCRIPTION
**Pull request by duokobia**
Here is my Week3 submission. I am having issues with setting the value of the input.

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -1,11 +1,30 @@
-import React from "react";
+import React, {useState} from "react";
+
+
+let AddTodoForm = ({onAddTodo}) => {
+
+  const[inputText, setInputText] = useState("");
```
Comment (kramduckner) [inline src/AddTodoForm.js]: I like how you're thinking! But I feel that an extra useState in this data flow could makes things confusing to maintain. I'd recommend not using this useState, and use the onAddTodo prop instead.

File: src/AddTodoForm.js [hunk 2]
```diff
@@ -1,11 +1,30 @@
...
+
 
-const AddTodoForm = () => {
   return (
-    <form>
+    <form onSubmit={(event) => handleAddTodo(event)}>
         <label htmlFor="todoTitle">Title</label>
-        <input id="todoTitle"></input>
-        <button>Add</button>
+        <input id="todoTitle" name="title" value = {inputText} onChange={(event) => setInputText(event.target.value)}></input>
```
Comment (kramduckner) [inline src/AddTodoForm.js]: While this is a valid way to get data out of an input, the instructions want us to only use an event handler on the form element. Feel free to remove the value and onChange prop on the input element

File: src/AddTodoForm.js [hunk 3]
```diff
@@ -1,11 +1,30 @@
...
+  
+
+  let handleAddTodo = function (event){
+    event.preventDefault(); // Prevent the default behaviour of the form submit.
+
+    // Retrieves the value of the title element from 
+    // the evenet target and store it in a variable named todoTitle.
+    let todoTitle = inputText; 
+    console.log(todoTitle);
+    setInputText(event.target.value);
```
Comment (kramduckner) [inline src/AddTodoForm.js]: You're so close to getting the value! In fact, I might have seen you correctly get it a commented out line. It'll be event.target.title.value.

**Review by kramduckner — CHANGES_REQUESTED** [review]
Nice job! I can tell you've been trying out different strategies to make it work. It shows you've been working hard and that you understand a lot of how React works. Keep up the awesome work! I'm confident you'll get it all working with some more practice.

File: src/AddTodoForm.js (hunk 1 above)
Comment (duokobia) [reply src/AddTodoForm.js]: Thanks for your comments Mark.

Noted.

File: src/AddTodoForm.js (hunk 2 above)
Comment (duokobia) [reply src/AddTodoForm.js]: I will do that.

Thanks.

File: src/AddTodoForm.js (hunk 3 above)
Comment (duokobia) [reply src/AddTodoForm.js]: Okay.

Thank you.

I will follow your recommendations and check if the app runs as expected.

I sincerely appreciate your efforts and time Mark.

Thank you so much.

**Comment by duokobia** [comment]
I just fixed it Mark. 

I appreciate your pointing me in the right direction.

Thanks a lot!

**Review by kramduckner — APPROVED** [review]
Nice work!! Approved. Feel free to merge!

**Pull request merged**